engine/api/src/main/scala/pl/touk/nussknacker/engine/api/process/RunMode.scala

```diff
@@ -0,0 +1,18 @@
+package pl.touk.nussknacker.engine.api.process
+
+/**
+  * Specifies the mode a custom node is invoked. It can be one of the following values:
+  * <ul>
+  * <li>Normal - process is executed inside Flink or other engine. Real implementation should be used.</li>
```

Comment:
Let's add comment that this mode is also used for validations, test data generations etc.